analyzers/src/SonarAnalyzer.CFG/ShimLayer/UnaryOperatorKind.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
```

Comment:
This one is copy/pasted and not autogenerated :D 

It will not help, as the whole project is excluded from analysis:
https://github.com/SonarSource/sonar-dotnet/blob/6bfeb6137b00d1fd5c6636c72fc3902c6ff26735/analyzers/src/SonarAnalyzer.ShimLayer.CodeGeneration/SonarAnalyzer.ShimLayer.CodeGeneration.csproj#L7